vinyl/hiddendelete.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+require_once "functions.php"; 
+
+$db = connectToDatabase(); 
+$data = fetchAllDatabase($db);
```

Comment:
There's actually no need to fetch the database here - all you need is the database connection on line 5.